experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -54,8 +54,46 @@ function Base.show(io::IO, G::LinearlyReductiveGroup)
 end
 
 #getter functions
-linearly_reductive_group(sym::Symbol, m::Int, R::MPolyRing) = LinearlyReductiveGroup(sym,m,R)
+
+@doc raw"""
+    linearly_reductive_group(sym::Symbol, m::Int, K::Field)
+
+If `sym` stands for implementing the special linear group, return SL$(m, K)$.
```

Comment:
This sentence is a bit hard to understand, it is not clear what "stands for implementing" means. Maybe
```
Return the linearly reductive group defined by `sym` over the field $K$ of degree $m$.

Currently, the supported options for `sym` are:
* `:SL` corresponding to the special linear group.
```

Reply:
Instead of "defined" I would go for something like "indicated" or so. Otherwise I agree with Johannes' suggestion.